.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+# https://github.com/semantic-release/semantic-release/blob/master/docs/recipes/github-actions.md
+name: Release
+on:
+  push:
+    branches:
+      - master
+jobs:
+  release:
+    name: Release
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
```

Comment:
Should the depth not be 1 as you only need the latest

Reply:
This is a release job, it needs all tag details to compute next version.
https://github.com/actions/checkout#fetch-all-history-for-all-tags-and-branches